akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/AccountExample10.java

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package jdocs.akka.persistence.typed;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.persistence.typed.PersistenceId;
+import akka.persistence.typed.javadsl.CommandHandler;
+import akka.persistence.typed.javadsl.CommandHandlerBuilder;
+import akka.persistence.typed.javadsl.EventHandler;
+import akka.persistence.typed.javadsl.EventHandlerBuilder;
+import akka.persistence.typed.javadsl.PersistentBehavior10;
+
+public class AccountExample10 extends PersistentBehavior10<AccountExample10.AccountCommand, AccountExample10.AccountEvent, AccountExample10.Account, AccountExample10.AccountSnapshot> {
```

Comment:
Can be compared with AccountExample https://github.com/akka/akka/blob/wip-persistenceId-patriknw/akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/AccountExample.java
which defines event handler outside the immutable state